src/ctap/mod.rs

```diff
@@ -2720,4 +2764,19 @@ mod test {
             ))
         );
     }
+
+    #[test]
```

Comment:
I would have put the `assert!` directly in the code, not as a test.
Otherwise nothing prevents me from configuring a wrong firmware and compiling it. I will never see the error unless I run `cargo test`.

Reply:
Assertion moved to code. Also I updated and merged the documentation. If we move to a configuration command, this property will probably be asserted there.

Reply:
Note that the current test still doesn't prevent from compiling. The key will just crash when the assert is executed. If we want static asserts we need a crate (it's still not supported by the language).

Reply:
True, but I decided to not introduce a new crate before we agreed on a long term plan for all constants. Also, if you make this invalid change after getting a pretty big warning in the documentation directly above that constant, this should be okay for now? Users' privacy should be sufficiently protected IMO.

Reply:
Yeah no problem, I was just mentioning it because the initial message from @jmichelp says:

> nothing prevents me from configuring a wrong firmware and compiling it

And this is still the case :D But yeah I hope if someone attempts to build a non-vanilla firmware, they would read the documentation surrounding their change.